Resources/Prototypes/Entities/Objects/Weapons/Guns/SMGs/smgs.yml

```
@@ -85,7 +85,7 @@
   name: C-20r submachine gun
   parent: [BaseWeaponSubMachineGun, BaseSyndicateContraband]
   id: WeaponSubMachineGunC20r
-  description: A firearm that is often used by the infamous nuclear operatives. Uses .35 auto ammo.
+  description: A Classic and widespread submachine gun, infamous for its use by The Gorlex Marauders. One of the first homegrown Waffle Corp. designs, the modernized ‘R’ variant remains in service today. Feeds from .35 SMG magazines.
```

```suggestion
  description: A classic and widespread submachine gun, infamous for its use by The Gorlex Marauders. One of the first homegrown Waffle Corp. designs, the modernized ‘R’ variant remains in service today. Feeds from .35 SMG magazines.
```

note: although cool, the description is extremely wordy, I would find ways to trim it

---

Will do

---

Updated the description, should be better now?